### PR TITLE
[CCR] Added auto_follow_exception.timestamp field to auto follow stats

### PR DESCRIPTION
**Pull request by martijnvg**
Currently auto follow stats users are unable to see whether an auto follow
error was recent or old. The new timestamp field will help user distinguish
between old and new errors.

### CODE REVIEW

File: x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java [hunk 1]
```diff
@@ -138,10 +138,11 @@ public synchronized AutoFollowStats getStats() {
     }
 
     synchronized void updateStats(List<AutoFollowResult> results) {
+        long newStatsReceivedTimeStamp = relativeMillisTimeProvider.getAsLong();
```
Comment (jasontedor) [inline x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java]: We want the absolute time provider here!

**Review by jasontedor — CHANGES_REQUESTED** [review]
We want a real clock instead of the relative clock, so that the timestamps are meaningful.

File: x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java (hunk 1 above)
Comment (martijnvg) [reply x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java]: whoops, you're absolutely right. I guess I made it this change too quickly...

**Review by jasontedor — APPROVED** [review]
LGTM.

**Pull request merged**